setup/helmRepository.yaml

```diff
@@ -10,4 +44,4 @@ spec:
   interval: 10m0s
   provider: generic
   type: oci
-  url: oci://ghcr.io/mirantis/2a-demos/charts
+  url: oci://helm-registry:5000/helm-charts
```

Comment:
I'm not sure if we should rather use a variable with a default of `oci://ghcr.io/mirantis/2a-demos/charts` here as this should only be used for local development, right? Or is the idea to abandon all charts currently stored in the GitHub repo as well?

Reply:
>Or is the idea to abandon all charts currently stored in the GitHub repo as well?

Yes, it should use only local registry with charts that are generated and pushed with the `make push-helm-charts` command (I added it to the README.md as well). Using Github repo as OCI registry is strictly bound us to the particular repo and can add various problems in the future if someone decides to fork this repo or create other charts/change existing